examples/react-todos/client/routes.jsx

```diff
@@ -39,7 +39,8 @@ Meteor.startup(function () {
       showFirstList();
     }
 
-    React.render(<Handler handles={ handles } />, document.body);
+    React.render(<Handler handles={ handles } />,
+      document.getElementById("app-container"));
```

Comment:
@avital one of the subtle issues with rendering directly into `document.body` is that 3rd party scripts (like analytics, social buttons, etc...) can inject nodes into `document.body` and React doesn't seem to like it. This can cause hard to track down bugs.